docs/getting-started/detections-req.asciidoc

```diff
@@ -42,11 +42,10 @@ and restarting {kib}, you must restart all detection rules.
 To enable the <<detection-engine-overview, Detections feature>>, a user with
 these privileges must visit (click on) the *Detections* page:
 
-* The `manage` cluster privilege
+* The `manage` cluster privilege.
 * {kib} space `All` privileges for the `Security` feature (see
 {kibana-ref}/xpack-spaces.html#spaces-control-user-access[Feature access based on user privileges]).
 * The `manage`, `write`,`read`, and `view_index_metadata` index privileges for all of these system indices:
-** `.siem-signals-<kib-space>`
```

Comment:
I think this is an accidental delete? We'll still need this along with `lists`/`items` right below.